src/atopile/parser/AtopileParser.g4

```diff
@@ -112,13 +107,11 @@ arithmetic_group
 literal_physical
     : bound_quantity
     | bilateral_quantity
-    | implicit_quantity;
+    | quantity;
 
-bound_quantity: quantity_end 'to' quantity_end;
-quantity_end: NUMBER name?;
-bilateral_quantity: bilateral_nominal PLUS_OR_MINUS bilateral_tolerance;
-implicit_quantity: NUMBER name?;
-bilateral_nominal: NUMBER name?;
+bound_quantity: quantity 'to' quantity;
```

Comment:
Do we want to make tolerances (especially the `to` operator) part of the nesting arithmetic expressions rather than a part of an atom